pom.xml

```diff
@@ -44,6 +44,17 @@
 		</developer>
 	</developers>
 
+	<repositories>
```

Comment:
This is not necessary

Reply:
True, a local change I forgot to roll back